packages/patternfly-3/patternfly-react/package.json

```diff
@@ -28,7 +28,7 @@
     "breakjs": "^1.0.0",
     "classnames": "^2.2.5",
     "css-element-queries": "^1.0.1",
-    "patternfly": "^3.52.4",
+    "patternfly": "^3.57.1",
     "react-bootstrap": "^0.32.1",
```

Comment:
You also need to update packages/patternfly-3/patternfly-react-extensions/package.json to use the same version